rust/src/lib.rs

```diff
@@ -128,6 +128,7 @@ mod ffi;
 pub mod architecture;
 pub mod backgroundtask;
 pub mod basicblock;
+pub mod binary_view_event_handler;
```

Comment:
Whatever this was named before was fine. If you're gonna name it like this, though, it would be better in `src/binaryview.rs` instead of its own file.

Reply:
ok, moving to src/binaryview.rs